internal/datastore/postgres/common/pgx.go

```diff
@@ -53,18 +54,46 @@ func queryTuples(ctx context.Context, sql string, args []any, span trace.Span, t
 			ResourceAndRelation: &corev1.ObjectAndRelation{},
 			Subject:             &corev1.ObjectAndRelation{},
 		}
-		err := rows.Scan(
-			&nextTuple.ResourceAndRelation.Namespace,
-			&nextTuple.ResourceAndRelation.ObjectId,
-			&nextTuple.ResourceAndRelation.Relation,
-			&nextTuple.Subject.Namespace,
-			&nextTuple.Subject.ObjectId,
-			&nextTuple.Subject.Relation,
-		)
+		var caveatName sql.NullString
+		var caveatCtx map[string]any
+		var err error
+		// TODO(vroldanbet) deprecate bool flag once CRDB also supports caveat
+		if caveatEnabled {
```

Comment:
We should always be able to read the data, even if the caveats are disabled, as the column still exists, right?

Reply:
no because the column does not exist in CRDB yet

Reply:
d'oh! ok